### PR TITLE
Add basic database structs needed

### PR DESCRIPTION
**Pull request by BharathTalakad**
- This merge commit adds basic structs need to quickly begin implementing essential endpoints like login, etc. 

- DBRelationships not written so need to update that

### CODE REVIEW

File: models/models.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	Line2   string
+	Pincode string
+}
+
+// User represents entity having all attributes required to represent an app user
+type User struct {
+	DBModel
+	FName    string
+	LName    string
+	UName    string
```
Comment (manas96agarwal) [inline models/models.go:28]: instead of UName just put username more clear to understand

File: models/models.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	Email    string
+	Vehicles string
```
Comment (manas96agarwal) [inline models/models.go]: What does Vehicles represent! is it only one vehicle or does it include multiple vehicles

File: models/models.go [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+	PhoneNo  string
+	Owner    Owner
```
Comment (manas96agarwal) [inline models/models.go:32]: How is Owner table related to User table

**Review by subodh101 — APPROVED** [review]
Good Start! We can make changes as we go on making new endpoints.
Overall LGTM!

**Review by subodh101 — APPROVED** [review]

**Pull request merged**